bedrock/firefox/tests/test_redirects.py

```diff
@@ -588,3 +568,203 @@ def test_offsite_redirects_still_work_when_locale_not_in_source_path(
+            "/en-US/firefox/android/releasenotes/",
+            "/en-US/firefox/android/releasenotes/?redirect_source=mozilla-org",
+        ),
+        (
+            "/en-US/firefox/android/releasenotes/?foo=bar",
+            "/en-US/firefox/android/releasenotes/?redirect_source=mozilla-org&foo=bar",
+        ),
+        (
+            "/en-US/firefox/ios/releasenotes/",
+            "/en-US/firefox/ios/releasenotes/?redirect_source=mozilla-org",
+        ),
+        (
+            "/en-US/firefox/ios/releasenotes/?foo=bar",
+            "/en-US/firefox/ios/releasenotes/?redirect_source=mozilla-org&foo=bar",
+        ),
+        (
+            "/en-US/firefox/nightly/notes/feed/",
+            "/en-US/firefox/nightly/notes/feed/?redirect_source=mozilla-org",
+        ),
+        (
+            "/en-US/firefox/nightly/notes/feed/?foo=bar",
+            "/en-US/firefox/nightly/notes/feed/?redirect_source=mozilla-org&foo=bar",
+        ),
+        (
+            "/en-US/firefox/android/system-requirements/",
+            "/en-US/firefox/android/system-requirements/?redirect_source=mozilla-org",
+        ),
+        (
+            "/en-US/firefox/android/system-requirements/?foo=bar",
+            "/en-US/firefox/android/system-requirements/?redirect_source=mozilla-org&foo=bar",
+        ),
+        (
+            "/en-US/firefox/system-requirements/",
+            "/en-US/firefox/system-requirements/?redirect_source=mozilla-org",
+        ),
+        (
+            "/en-US/firefox/system-requirements/?foo=bar",
+            "/en-US/firefox/system-requirements/?redirect_source=mozilla-org&foo=bar",
+        ),
+        (
+            "/en-US/firefox/ios/system-requirements/",
+            "/en-US/firefox/ios/system-requirements/?redirect_source=mozilla-org",
+        ),
+        (
+            "/en-US/firefox/ios/system-requirements/?foo=bar",
+            "/en-US/firefox/ios/system-requirements/?redirect_source=mozilla-org&foo=bar",
+        ),
+        (
+            "/en-US/firefox/android/system-requirements/",
+            "/en-US/firefox/android/system-requirements/?redirect_source=mozilla-org",
+        ),
+        (
+            "/en-US/firefox/android/system-requirements/?foo=bar",
+            "/en-US/firefox/android/system-requirements/?redirect_source=mozilla-org&foo=bar",
+        ),
+        (
+            "/en-US/firefox/system-requirements/",
+            "/en-US/firefox/system-requirements/?redirect_source=mozilla-org",
+        ),
+        (
+            "/en-US/firefox/system-requirements/?foo=bar",
+            "/en-US/firefox/system-requirements/?redirect_source=mozilla-org&foo=bar",
+        ),
+        (
+            "/en-US/firefox/ios/system-requirements/",
+            "/en-US/firefox/ios/system-requirements/?redirect_source=mozilla-org",
+        ),
+        (
+            "/en-US/firefox/ios/system-requirements/?foo=bar",
+            "/en-US/firefox/ios/system-requirements/?redirect_source=mozilla-org&foo=bar",
```

Comment:
these also appear to be tested twice, weren't they supposed to be something else, or are just copypasta extra?